modules/candidate_list/templates/menu_candidate_list.tpl

```diff
@@ -1,4 +1,24 @@
-<div class="row">
+<script type="text/javascript" src="js/advancedMenu.js"></script>
+{literal}
+<script type="text/javascript">
+    $(document).ready(function(){
+        var colm_static = false;
+        $(".table-scroll").scroll(function(){
+            if(colm_static === true){
+                if($(".colm-site").offset().left >= 30){
+                    $(".colm-pscid").removeClass("static-col colm-static");
+                    colm_static = false;
+                }
+            } else if($(".colm-pscid").offset().left <= 35){
+                $(".colm-pscid").addClass("static-col colm-static");
+                colm_static = true;
+            }
+        });
```

Comment:
Why is this here? If it's required for the freezing, shouldn't it be done in the plugin?